org.kde.skrooge.yml

```diff
@@ -0,0 +1,70 @@
+id: org.kde.skrooge
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.11'
+command: skrooge
+rename-icon: skrooge
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+cleanup:
```

Comment:
You need to manually delete the baseapp files with `cleanup-commands` I believe.

Reply:
I think `cleanup` also handles it fine.